BackEndApp/v1/views.py

```diff
@@ -13,10 +14,12 @@
 from v1.models import UserData
 from dlmml.utils import json_to_dict
 from dlmml.parser import *
-
+from dotenv import load_dotenv
```

Comment:
Is this needed?

Reply:
Ahh nope this isn't needed (previously was - b4 the separate django app).... I'll remove it right away xD

Reply:
@ADI10HERO I've removed the unnecessary `dotenv` lines in the latest commit to the PR